.github/CONTRIBUTING.md

```diff
@@ -3,6 +3,23 @@
 This document is intended to answer questions from (potentially new) contributors for all.
 Do you have a question which is not answered here? Feel free to [open an issue](https://github.com/CoRoSoN-Kit/CoRoSoN-Kit/issues/new) about it.
 
+## Repo/Folder structure
+
+```text
+coroson-kit
```

Comment:
In one of our first meetings we already talked about folder structure, at that time I think we agreed to distinguish by components on the top level. 

So first folder would be `brain` for example, like I did in this draft: #46

What do you think @Alintya ?

Reply:
I would really like to have src and hw on top of the components in order to not clutter up the top level as much as it would be much harder no navigate imo
Also, having code files near each other prevents having horrendous reference paths

Reply:
Ok, that fits for me too. But then we should create separate libraries folders, so one under src and one under hw (to avoid the many ../.. links)

Reply:
Agreed